application/src/main/java/com/speedment/jpastreamer/application/JPAStreamer.java

```diff
@@ -25,7 +25,10 @@
 import java.util.stream.Stream;
 
 /**
- * A JPAStreamer is responsible for creating Streams from data sources
+ * A JPAStreamer is responsible for creating Streams from data sources,
+ * alternatively for creating Streamers that can be reused to create Streams of the same Entity source, 
```

Comment:
creating `StreamSuppliers`

Reply:
Thanks, a relic of my first attempt to introduce Streamers.